src/components/gui/gui.css

```diff
@@ -21,24 +21,71 @@
     height: 100%;
 }
 
-.blocks-wrapper {
-    /*
-        scratch-blocks is based on absolute positioning, which injects
-        inside this element and becomes the child
-    */
+.editor-wrapper {
+    flex-basis: 600px;
+    flex-grow: 1;
+    flex-shrink: 0;
     position: relative;
 
-    flex-basis: 600px;
+    display: flex;
+    flex-direction: column;
+}
+
+.tab-list {
+    height: $stage-menu-height;
+    width: 250px; /* Match width of the toolbox */
+    display: flex;
+    align-items: flex-end;
+    flex-shrink: 0;
+
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-weight: 500;
+    font-size: 0.80rem;
+
+    /* Overrides for react-tabs styling */
+    margin: 0 !important;
+    border-bottom: 0 !important;
+}
+
+.tab-list .tab {
+    flex-grow: 1;
+    height: 80%;
+    margin-left: 1px;
+
+    border-radius: $space $space 0 0;
+    border: none;
+
+    background-color: #F6F8FA;
+    color: #9AA1B5;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+
+.tab-list .tab[aria-selected="true"] {
+    color: #40B9F5;
+}
+
+.tabs {
+    position: relative;
     flex-grow: 1;
     flex-shrink: 0;
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
Having these styles here is fine for now, but it would be nice to eventually pull these styles out and put them into our own tab components (which can just wrap the `react-tabs` library components).